workflows/nanostring.nf

```diff
@@ -83,31 +87,26 @@ workflow NANOSTRING {
         rcc_files,
         samplesheet_path
     )
-    ch_versions = ch_versions.mix(NORMALIZE.out.versions)
-
+    ch_versions         = ch_versions.mix(NORMALIZE.out.versions)
+    ch_normalized       = NORMALIZE.out.normalized_counts
+    ch_normalized_wo_hk = NORMALIZE.out.normalized_counts_wo_HK
 
     //
     // MODULE: Annotate normalized counts with metadata from the samplesheet
     //
     CREATE_ANNOTATED_TABLES (
-        NORMALIZE.out.normalized_counts.mix(NORMALIZE.out.normalized_counts_wo_HK).toSortedList().flatten(),
+        ch_normalized.mix(ch_normalized_wo_hk).toSortedList{ a, b -> a[1].getName() <=> b[1].getName() }.flatMap(), // Order tuples (based on file name) to stabilize tests and reproducibility
```

Comment:
Same here, I would prefer we use join, even though it doesn't affect the current state of the pipeline, it may cause issues in the future.